packages/mdc-floating-label/test/component.test.ts

```diff
@@ -22,60 +22,70 @@
  */
 
 
-import {MDCNotchedOutline} from '../../mdc-notched-outline/index';
```

Comment:
it this `git copy` automatically created by Copybara? 🤔

Reply:
Hmm yes, this is strange

Reply:
I think this PR accidentally replaced the notchedoutline test file. Can we restore it?

Reply:
Ah sorry, must've missed a sync - opened #5449